spring-kafka/src/main/java/org/springframework/kafka/annotation/KafkaListener.java

```diff
@@ -297,4 +297,31 @@
 	 */
 	String batch() default "";
 
+	/**
+	 * Set an {@link org.springframework.kafka.listener.adapter.RecordFilterStrategy} bean
+	 * name to override the strategy configured on the container factory. If a SpEL
+	 * expression is provided ({@code #{...}}), the expression can either evaluate to a
+	 * {@link org.springframework.kafka.listener.adapter.RecordFilterStrategy} instance or
+	 * a bean name.
+	 * @return the error handler.
+	 * @since 2.8.4
+	 */
+	String filter() default "";
+
+	/**
+	 * Static information that will be added as a header with key
+	 * {@link org.springframework.kafka.support.KafkaHeaders#LISTENER_INFO}. This can be used, for example, in a
+	 * {@link org.springframework.kafka.listener.RecordInterceptor}, {@link RecordFiorg.springframework.kafka.listener.adapter.RecordFilterStrategylterStrategy} or the listener itself, for
```

Comment:
Typo for the `RecordFilterStrategy`:
```suggestion
	 * {@link org.springframework.kafka.listener.RecordInterceptor}, {@link org.springframework.kafka.listener.adapter.RecordFilterStrategy} or the listener itself, for
```